code/script.js

```diff
@@ -241,103 +243,152 @@ const generateBoard = () => {
         </div>
       </div>
     `
-  })
+    })
 }
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+    secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
+
+
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+    charactersInPlay = CHARACTERS
+    board.style.display = 'flex'
+    winOrLose.style.display = 'none'
+    generateBoard()
+    setSecret()
 }
 
+
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
+    const category = questions.options[questions.selectedIndex].parentNode.label
+    const selectedValue = questions.options[questions.selectedIndex].value
 
-  if (category === 'hair color') {
-    currentQuestion = {
-      attribute: 'hairColor',
-      // value: ,
-      // 👆 add the value from the input here
-      category: category,
+    /* the variable currentQuestion is created as an object to use for comparison with the "secret"-person*/
+    if (category === 'hair color') {
+        currentQuestion = {
+            attribute: 'hairColor',
+            value: selectedValue,
+            category: category,
+        }
```

Comment:
When it's the same attribute as value, you could just write category, :)

Reply:
OK, cool! I didn't know that.